src/_locales/zh_TW/messages.json

```diff
@@ -1,496 +1,461 @@
-    "hash": "344e853acdaef0ac95e161cdbc201036"
+    "message": "Q 級（25%）",
+    "description": "Option of error correction level for QR code. See https://en.wikipedia.org/wiki/QR_code#Error_correction."
   },
   "optionEcHigh": {
-    "message": "高(30%)",
-    "description": "Option of error correction level for QR code. See https://en.wikipedia.org/wiki/QR_code#Error_correction.",
-    "hash": "30035374d52e135473d464fe1726ad8c"
+    "message": "H 級（30%）",
+    "description": "Option of error correction level for QR code. See https://en.wikipedia.org/wiki/QR_code#Error_correction."
   },
   "optionErrorCorrectionDescr": {
-    "message": "這個值表示可以丟失多少碼塊而不影響掃描。",
-    "description": "The description of the error correction option.",
-    "hash": "2bbbd8aafc044a910d26cbf04e560766"
+    "message": "此值表示遺失（掃不到）多少碼塊，也可以讀到內容。",
+    "description": "The description of the error correction option."
   },
   "optionErrorCorrectionDescrLink": {
     "message": "https://en.wikipedia.org/wiki/QR_code#Error_correction",
-    "description": "The 'Learn more' link in the description of the error correction option.",
-    "hash": "530f6b1ee177dddd7d40571d98a64408"
+    "description": "The 'Learn more' link in the description of the error correction option."
   },
+
   "optionAutoGetSelectedText": {
-    "message": "自動使用網站上選取的文字",
-    "description": "This is an option shown in the add-on settings.",
-    "hash": "21667c5a30f89561cea02351172b9c91"
+    "message": "自動使用網頁中選取的文字",
+    "description": "This is an option shown in the add-on settings."
   },
   "optionUseMonospaceFont": {
     "message": "使用等寬字體",
-    "description": "This is an option shown in the add-on settings.",
-    "hash": "69191273c436064be8c4f5d6a8f675f5"
+    "description": "This is an option shown in the add-on settings."
+  },
+
+  "optionAutoGetClipboardContent": {
+    "message": "開起彈出視窗時使用剪貼簿的內容",
+    "description": "This is an option shown in the add-on settings."
+  },
+  "buttonRequestPermission": {
+    "message": "授予權限",
+    "description": "The button label, used for requesting a permission that is missing."
+  },
+  "couldNotRequestPermission": {
+    "message": "權限請求失敗。",
+    "description": "When the permission request fails."
   },
+
   "optionDebugMode": {
-    "message": "啟用除錯模式",
-    "description": "This is an option shown in the add-on settings.",
-    "hash": "de6506af4973e0980d64f242fba11a19"
+    "message": "啟用偵錯模式",
+    "description": "This is an option shown in the add-on settings."
   },
   "optionDebugModeDescr": {
-    "message": "僅用在控制台取得更詳細的日誌輸出用於匯報問題...等等。",
-    "description": "The description shown for the debug mode (optionDebugMode).",
-    "hash": "2aff10770796694827ec4253860e0771"
+    "message": "只有在控制臺取得更詳細的日誌輸出來匯報錯誤等情況時有用。",
+    "description": "The description shown for the debug mode (optionDebugMode)."
   },
+
   "translatorCredit": {
```

Comment:
As for how to refer to all contributors. 

You can credit yourself here and provide a link (in `translatorLink`) and a name (in `translatorUsername`). Also, you should add yourself to the CONTRIBUTORS.md file and the changelog as a translator to [give credits here](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#credit).
You can just provide `translatorUsername` for the link text aka all contributors or a generic term and then adjust `translatorLink` to link to https://github.com/rugk/offline-qr-code/blob/master/CONTRIBUTORS.md#chinese-simplified. Aka adjust that Markdown file to list all contributors, that would be nice.

See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translations for how these placeholders are translated, especially [the official doc](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/Internationalization#Providing_localized_strings_in__locales).